reactor-test/src/main/java/reactor/test/DefaultStepVerifierBuilder.java

```diff
@@ -79,6 +79,51 @@
 	 */
 	static Duration defaultVerifyTimeout = StepVerifier.DEFAULT_VERIFY_TIMEOUT;
 
+	private static class StreamStepRecorder {
```

Comment:
Can you rename to `HookRecorder`? (even if it is `private`, I think it is more fitting than `StreamStep`).